### DISCUSSION TITLE
请教下能否使用其它字幕渲染引擎

### DISCUSSION THREAD

**Discussion by Moyikc** [hooke007/mpv_PlayKit#510] [mpv 通用]
有一个ass字幕用了vsfiltermod的渐变色特效代码，感觉效果还不错，要是用vsfilter原版或者libass的要用aegisub的lua脚本实现，麻烦些。所以想问问mpv能改成用vsfiltermod渲染字幕吗？
搜了一下，看到 [这个讨论：讨论多重高级字幕并载](https://github.com/hooke007/MPV_lazy/discussions/61) 里有提到似乎可以用VapourSynth间接实现？之前没用过VapourSynth，了解了下是通过vpy文件调用的，有没有用vsfiltermod的示例代码呢

**Comment by hooke007** [comment]
不能。

vs没有实际意义，时间轴无法同步

**Comment by hooke007** [comment]
vsm做的特效应该硬编或者生成图形字幕，做成ass属于自嗨的行为，没有普适价值（没有其他程序和硬件平台兼容它自创的规范）。
